fix(parsers, KR): fix broken consumption parsing, and add wind power

consideRatio opened the pull request:
## Issue

- Fixes #5469
  There were some HTML changes making the consumption parser fail consistently, which the title of the issue doesn't reflect.
- Fixes #7682

### Double check

- [x] I have tested my parser changes locally with `poetry run test_parser "zone_key"`
  ```
  # likely requires WEBSHARE_USERNAME / WEBSHARE_PASSWORD environment variables to be set
  poetry run test_parser "KR" production
  poetry run test_parser "KR" consumption
  poetry run test_parser "KR" price
  ```
- [x] I have run `pnpx prettier@2 --write .` and `poetry run format` in the top level directory to format my changes.

VIKTORVAV99 commented:
This is great and I'll review it tomorrow but I found out the KR parser is down because we used up all our bandwidth for the proxy, I'll upgrade it as soon as possible!

consideRatio commented:
Test failures seems unrelated, I opened #7689

VIKTORVAV99 commented on poetry.lock:
Do you think we can make do without these changes?

It just takes forever to deploy when there is dependency changes as all the caches are invalidated.

consideRatio replied:
:+1: included by mistake debugging python test failures!

VIKTORVAV99 commented on pyproject.toml:
```diff
@@ -83,6 +83,8 @@ click = "^8.0.0"
 
 [tool.pytest.ini_options]
 testpaths = ["tests", "parsers/test", "electricitymap/contrib/lib/tests"]
+addopts = "--verbose --color=yes --durations=10"
+
```
What does this do? 👀

consideRatio replied:
Ooops, mistakenly included in this PR.

VIKTORVAV99 reviewed (commented):
Parser looks good but two minor comments to be addressed before we can merge 🙂

VIKTORVAV99 reviewed (approved):
LGTM! 🎉

consideRatio commented:
I mistakenly included unrelated commits into this PR - I removed those commits and rebased on master and force-pushed. Anything with KR parser etc hasn't been adjusted since you last looked at this PR @VIKTORVAV99.

Thanks for finding time to review things!!!
